modules/n1ql/examples/select/index-hint.jsonc

```diff
@@ -1,119 +1,123 @@
-{
-  "cardinality": 1,
-  "cost": 25.48328348473815,
-  "optimizer_hints": {
-    "hints_followed": [
-      "INDEX(route def_inventory_route_route_src_dst_day)"
-    ]
-  },
-  "plan": {
-    "#operator": "Sequence",
-    "~children": [
-      {
-        "#operator": "Sequence",
-        "~children": [
-          {
-            "#operator": "DistinctScan",
-            "limit": "1",
-            "optimizer_estimates": {
-              "cardinality": 1681.2723543436066,
-              "cost": 395.71744847285015,
-              "fr_cost": 12.220147704229976,
-              "size": 11
-            },
+[
+  {
+    "cardinality": 1,
+    "cost": 30.946612007857688,
+    "optimizer_hints": {
+      "hints_followed": [
+        "INDEX(route def_samples_route_route_src_dst_day)"
+      ]
+    },
+    "plan": {
+      "#operator": "Sequence",
+      "~children": [
+        {
+          "#operator": "Sequence",
+          "~children": [
+            {
+              "#operator": "DistinctScan",
+              "limit": "1",
+              "optimizer_estimates": {
+                "cardinality": 249,
+                "cost": 9928.010826065869,
+                "fr_cost": 17.689048092980993,
+                "size": 11
+              },
 // tag::index[]
-            "scan": {
-              "#operator": "IndexScan3",
-              "bucket": "travel-sample",
-              "index": "def_inventory_route_route_src_dst_day",
+              "scan": {
+                "#operator": "IndexScan3",
+                "bucket": "my-db-7of81g",
```

Comment:
Ah right - so the difference you see there is because there is a "real" bucket name that the db actually uses, and then theres the one the user actually sees (although there are still cases where the user sees the "real" name). For the simple results I just subbed in the my-db name, but for the more complex ones where I had to actually run the queries the result is the "real" name. There was a comment on the elixir user forum channel this morning complaining about this very issue - the user really shouldn't be able to see both.

Reply:
Eep. That's not good at all. 

What I was getting at was the following text, the `my-db-7of81g` is different from the `my-db-p4ox4` or whatever it was elsewhere. Do we want this to be different?

Reply:
That difference is because I had to make about 20 dbs because they kept crashing. I don't think it'll make a noticeable difference

Reply:
Humour me and make it the same? :) Or just change it to a clear placeholder value where it occurs. Users might not be reading this sequentially but we want to keep consistency in our examples.

Reply:
👍 done